### PR TITLE
Vrbranch micrometrics ratpack

### PR DESCRIPTION
**Pull request by vijaycse**
Closes the issue  https://github.com/resilience4j/resilience4j/issues/1326

### CODE REVIEW

**Comment by vijaycse** [comment]
@drmaas  Can you take a look  at this PR?

File: resilience4j-ratpack/build.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@ dependencies {
     compileOnly(libraries.ratpack)
     compile project(':resilience4j-annotations')
     compile project(':resilience4j-framework-common')
+    compileOnly(libraries.micrometer)
```
Comment (vijaycse) [inline resilience4j-ratpack/build.gradle:7]: Not sure why `resilience4j-micrometer` is not bringing micormeter core transitively. had to add it explicitly

**Comment by vijaycse** [comment]
After discussing with @drmaas we decided to hold off this change until raptack is ready with ratpack module.
then we need to upgrade ratpack version in resilence4j before mering.

**Comment by Romeh** [comment]
So should we close that PR for now @vijaycse  ?

**Comment by vijaycse** [comment]
Sure. I can open it again. will close for now @Romeh

**Pull request closed without merging**